rtc_types/src/attestation.rs

```diff
@@ -0,0 +1,26 @@
+use std::vec::Vec;
```

Comment:
Since we don't need these types on the enclave side, I there is no need for us to add it to the `rtc_types`. `rtc_types` is used to define types we need on the untrusted, and trusted side for the most part.